src/main/java/com/revature/services/JSONReaderService.java

```diff
@@ -1,136 +0,0 @@
-package com.revature.services;
```

Comment:
deleting this file was a mistake. I am pretty sure we can safely delete it... but plan on doing more testing first. (I was testing the theory that this file is exclusively a debug tool used before they had a functional database that is now redundant. I must have missed this deletion on one of the patch commits I submitted.